client/src/modules/CourseManagement/components/CourseEventModal/formState.ts

```diff
@@ -83,8 +83,8 @@ export function getInitialValues(modalData: Partial<CourseEvent>) {
     type: EVENT_TYPES.find(event => event.id === modalData.event?.type)?.id ?? null,
     descriptionUrl: modalData.event?.descriptionUrl ? modalData.event.descriptionUrl : '',
     description: modalData.event?.description ? modalData.event.description : '',
-    dateTime: modalData.dateTime ? dayjs.utc(modalData.dateTime) : null,
-    endTime: modalData.endTime ? dayjs.utc(modalData.endTime) : null,
+    dateTime: dayjs.utc(modalData.dateTime),
```

Comment:
why? if the dateTime is `null` you would get invalid date, which might cause bugs in other places

Reply:
thank you, i fix this problem.
I've created an issue for refactoring and will address it as soon as possible. #2342